ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -222,92 +199,113 @@ class TransactionSummaryScreen extends React.Component<Props> {
     );
   }
 
+  private getFooterButtons = () => {
+    return this.props.error.fold(
+      this.renderFooterButtons(),
+      error =>
+        error === "PAYMENT_DUPLICATED"
+          ? this.renderFooterSingleButton()
+          : this.renderFooterButtons()
+    );
+  };
+
   public render(): React.ReactNode {
-    const rptId = this.props.navigation.getParam("rptId");
-    const initialAmount = this.props.navigation.getParam("initialAmount");
+    const rptId: RptId = this.props.navigation.getParam("rptId");
+    // TODO: it should compare the current an d the initial amount BUT the initialAmount seems to be provided with an incorrect format https://www.pivotaltracker.com/story/show/172084929
+    const isAmountUpdated = true;
 
-    // when empty, it means we're still loading the verifica response
     const { potVerifica } = this.props;
 
-    const maybeEnteBeneficiario = pot
+    const recipient = pot
       .toOption(potVerifica)
       .mapNullable(_ => _.enteBeneficiario)
       .map(formatTextRecipient);
 
+    const currentAmount: string = pot.getOrElse(
+      pot.map(potVerifica, (verifica: PaymentRequestsGetResponse) =>
+        formatNumberAmount(centsToAmount(verifica.importoSingoloVersamento))
+      ),
+      "-"
+    );
+
+    const transactionDescription =
```

Comment:
```suggestion
   const transactionDescription = pot.getOrElse<string>(
      pot.mapNullable(potVerifica, pv => pv.causaleVersamento),
      "-"
    );
```
ps: I didn't test it

Reply:
updated wihtin  https://github.com/pagopa/io-app/pull/1628/commits/0bb43f2e3863593914119b29706a04c1f935e3b3. It was missing just the function correcting the string format